PbiPaasWebApi/packages.config

```diff
@@ -12,13 +12,17 @@
   <package id="Microsoft.AspNet.WebApi.HelpPage" version="5.2.3" targetFramework="net452" />
   <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.3" targetFramework="net452" />
   <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net452" />
-  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.0" targetFramework="net452" />
-  <package id="Microsoft.Net.Compilers" version="1.0.0" targetFramework="net452" developmentDependency="true" />
-  <package id="Microsoft.PowerBI.Api" version="1.0.7-preview" targetFramework="net452" />
-  <package id="Microsoft.PowerBI.Core" version="1.0.7-preview" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.0.1" targetFramework="net452" />
+  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.1" targetFramework="net452" />
+  <package id="Microsoft.IdentityModel.Logging" version="1.0.0" targetFramework="net452" />
+  <package id="Microsoft.IdentityModel.Tokens" version="5.0.0" targetFramework="net452" />
+  <package id="Microsoft.Net.Compilers" version="1.3.2" targetFramework="net452" developmentDependency="true" />
+  <package id="Microsoft.PowerBI.Api" version="1.1.1" targetFramework="net452" />
+  <package id="Microsoft.PowerBI.AspNet.Mvc" version="1.1.1" targetFramework="net452" />
```

Comment:
Microsoft.PowerBI.AspNet.Mvc is not needed. This package adds Html helpers to facilitate embedding which do not have a purpose for REST API service.